packages/patternfly-4/react-core/src/components/Modal/Modal.js

```diff
@@ -98,15 +98,15 @@ class Modal extends React.Component {
   render() {
     const { ...props } = this.props;
 
-    if (!canUseDOM) {
+    if (!ExecutionEnvironment.canUseDOM) {
       return null;
     }
 
     if (!this.container) {
```

Comment:
Move the lines that create the container to the onComponentMount() lifecycle method.